### DISCUSSION TITLE
I haven't received an email regarding my student pack application in over a month

### DISCUSSION THREAD

**Discussion by lMartin3** [community/community#23541] [GitHub Education]
<p>Hello, on the 12th of November, I have applied for the GitHub student pack, and I have received this email the same day:<br>
</p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/a/c/ace58fc350e19078f66826a1abe9e6b970443815.png" data-download-href="/uploads/short-url/oFvXpMNsaVTCEQ06Ts3soiJxyx7.png?dl=1" title="ged" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/38576621/181077809-6fcf6166-f409-4ba1-b100-8e483842449e.png" alt="ged" data-base62-sha1="oFvXpMNsaVTCEQ06Ts3soiJxyx7" width="565" height="500" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/a/c/ace58fc350e19078f66826a1abe9e6b970443815_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">ged</span><span class="informations">657×581 16.3 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div><br>
I still haven’t received any emails regarding my application, what should I do now?<br>
Thanks.

**Comment by canuckjacq** [comment, marked as answer]
<p>:wave: Welcome!</p>
<p>Sorry you didn’t get an email but it looks like you’ve been approved.</p>
<p>You can access your GitHub Education benefits here:</p>
<p><a href="https://education.github.com/benefits">https://education.github.com/benefits </a></p>
<p>Some of our partner offers are single-use and non-renewable. Most timed offers begin when redeemed, not when granted.</p>
<p>If you have questions or issues with an offer, please contact the partner directly by clicking the <img src="https://user-images.githubusercontent.com/11161530/181077810-a8857c58-73d2-41ec-880c-cc0685628e56.png" alt="help available at" data-base62-sha1="7qn7mg2Z1Fn22zxR6OdKoVsJ3YP" width="123" height="22"> link on the offer tile:</p>
<p><a href="https://education.github.com/pack/offers" class="onebox" target="_blank" rel="noopener">https://education.github.com/pack/offers</a></p>

**Comment by lMartin3** [comment]
<p>Thanks! I can access the benefits without any issues.</p>

**Comment by nibin-matteo-git** [comment]
<p>HI!<br>
I too have the same issue.<br>
But i just found out that i got approved (but didnt get the email) by clicking on the above link.<br>
</p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/f/f/ff999973da9110c00ff7fe9a0a0ebf26e7b8fb5b.jpeg" data-download-href="/uploads/short-url/At8Q5MeqRSae8Zm4LuNwwu3qNB1.jpeg?dl=1" title="Screenshot 2021-04-17 at 8.42.48 AM" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/80332406/181077812-60e5efe5-af3b-4040-98fa-e8df6f71d5f3.jpeg" alt="Screenshot 2021-04-17 at 8.42.48 AM" data-base62-sha1="At8Q5MeqRSae8Zm4LuNwwu3qNB1" width="690" height="431" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/f/f/ff999973da9110c00ff7fe9a0a0ebf26e7b8fb5b_2_690x431.jpeg, https://aws1.discourse-cdn.com/github/optimized/3X/f/f/ff999973da9110c00ff7fe9a0a0ebf26e7b8fb5b_2_1035x646.jpeg 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/f/f/ff999973da9110c00ff7fe9a0a0ebf26e7b8fb5b_2_1380x862.jpeg 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/f/f/ff999973da9110c00ff7fe9a0a0ebf26e7b8fb5b_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Screenshot 2021-04-17 at 8.42.48 AM</span><span class="informations">2880×1800 427 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>But I don’t know how to avail it.<br>
I tried getting the pycharm pro and it doesnt let me even after i log in using my github account.<br>
A little help would be great.</p>

**Comment by canuckjacq** [comment]
<p><a class="mention" href="/nibin-matteo-git">@nibin-matteo-git</a> Sorry you didn’t get your email.</p>
<p>You can access your offers here: <a href="https://education.github.com/pack/offers">https://education.github.com/pack/offers</a></p>
<p>For Jetbrains, this looks like the form you need: <a href="https://www.jetbrains.com/shop/eform/students" class="inline-onebox">JetBrains Products for Learning</a> (just pick the GitHub option).</p>

**Comment by atanughosh01** [comment]
<p>Although I enrolled in the GitHub student developer program, I still can’t access any <a href="http://educative.io" rel="noopener nofollow ugc">educative.io</a> courses, why?</p>

**Comment by canuckjacq** [comment]
<p>:wave: Welcome!</p>
<p>If you’re unable to verify your student status on <a href="https://www.educative.io/github-students">their website</a> you may have to contact their support to find out why.</p>

**Comment by Sumant03** [comment]
<p>I haven’t received any email for Github Campus Ambassador what should I do</p>

**Comment by kalpan-shah** [comment]
<p>My req for the student pack is also approved but I didn’t get a verification code needed for the MS Azure subscription.</p>

**Comment by csakshi07** [comment]
<p>HI!<br>
I too have the same issue.<br>
But i just found out that i got approved (but didnt get the email) .<br>
</p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/1/d/1d64bc8ee0b27a79e05b7dc2e2b048e05072215c.png" data-download-href="/uploads/short-url/4c1FLLDbdYythrF3D3gJODPptTe.png?dl=1" title="Screenshot (455)" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/91685608/181077820-153bb002-50ef-4458-8e62-a6cd1312bb3c.png" alt="Screenshot (455)" data-base62-sha1="4c1FLLDbdYythrF3D3gJODPptTe" width="690" height="483" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/1/d/1d64bc8ee0b27a79e05b7dc2e2b048e05072215c_2_690x483.png, https://aws1.discourse-cdn.com/github/original/3X/1/d/1d64bc8ee0b27a79e05b7dc2e2b048e05072215c.png 1.5x, https://aws1.discourse-cdn.com/github/original/3X/1/d/1d64bc8ee0b27a79e05b7dc2e2b048e05072215c.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/1/d/1d64bc8ee0b27a79e05b7dc2e2b048e05072215c_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">Screenshot (455)</span><span class="informations">998×699 57 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by 56628426** [comment]
Guys what's really use of GitHub

**Reply by asim1909** [reply]
![github](https://user-images.githubusercontent.com/118390636/213874081-5000ed5a-b471-44c6-a7c4-cb96838151c8.jpg)

**Comment by Manjeet-Agarwal** [comment]
Hii, 
I am also having the same issue, My GitHub Student Developer Pack Application has been approved but I still did not receive any E-Mail to access the benefits!

Can anyone tell how much it will take as 5 Days have already been completed since approval!!

<img width="856" alt="Github Student Developer Pack Application Approved" src="https://github.com/community/community/assets/114595496/67b63f55-6a83-40c5-b92e-ac5045962291">

**Reply by Manjeet-Agarwal** [reply]
Finally, I have received my GitHub Studnet Developer pack email.

**Reply by aum99** [reply]
how?

**Reply by AliyahM-12** [reply]
How long did it take for you to receive the email??

**Comment by ertugrulemrecihan** [comment]
Hi, I'm having the same issue, my GitHub Student Developer Pack Application has been approved but I still haven't received any Email to access the benefits!

It's Been 2 Days Since Approval And I Still Haven't Received An Email This Is Very Annoying

![image](https://github.com/community/community/assets/64169298/9c79600d-7a79-422b-8108-94b782d7c8ac)

**Comment by proappple123** [comment]
Same, still didn't receive any email but it shows that I'm approved already.

**Comment by DawnHNguyen** [comment]
Me too, my GitHub Student Developer Pack Application has been approved but I still couldn't access copilot. It's already 3 days.

**Comment by zffs** [comment]
I also got approved for the GitHub Student Developer Pack  but haven't received the benefits yet.

**Comment by decoderanu11** [comment]
Same issue with me
It's been 3 days approved for student account
But not received any student benifits yet.

**Comment by ghost** [comment]
When applying for the github student package, the iPhone’s rear camera cannot be used, and the photos taken with Android are not clear. Who has encountered this problem? Can’t add a photo that can be uploaded in the album?

**Comment by TTrincado** [comment]
Same issue, already approved but still can't access to the benefits.. :(

**Reply by samus-aran** [reply]
Hi @TTrincado

We can see that your academic benefits have been granted, congratulations! Enjoy! 🎉

**Comment by samus-aran** [comment]
We will be closing this thread as the verification process has been modified since 2020 so many of these comments are not relevant to the current process.
